.github/workflows/build.yml

```diff
@@ -0,0 +1,16 @@
+name: Build test
+on:
+  push:
+    branches:
+      - main
+jobs:
+  install-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install npm dependencies
+        run: |
+          npm install
+      - name: Run Linter
+        run: |
+          npm run lint
```

Comment:
So currently we only consider the linting issues. My suggestion is to hold this until we have some test that can be executed during the build. This action will not actually reflect the requirement of a builder. @Wathsara need some clarification on this.

Reply:
@shehand Since we dont have any test cases and @AbhiKul10 still works on that, hope this initial step is fine. Once we have tests We can update the action.

Reply:
Yes @shehand ! I am on it and I'll update the actions too for the same.